### PR TITLE
PERF: removes two unnecessary enclosing divs for each message

### PR DESCRIPTION
**Pull request by jjaffeux**

### CODE REVIEW

**Comment by jjaffeux** [comment]
This will make the DOM structure of chat messages much cleaner.

File: assets/javascripts/discourse/components/chat-message.js [hunk 1]
```diff
@@ -528,11 +539,15 @@ export default Component.extend({
   },
 
   _repositionEmojiPicker(btn, position) {
-    if (!this.element) {
+    const element = document.querySelector(
+      `.chat-message-container[data-id='${this.message.id}']`
+    );
```
Comment (CvX) [inline assets/javascripts/discourse/components/chat-message.js:544]: Maybe this should be extracted to `get element() {}`?

Comment (jjaffeux) [reply assets/javascripts/discourse/components/chat-message.js:544]: ok :+1:

File: assets/javascripts/discourse/components/chat-message.js [hunk 2]
```diff
@@ -43,11 +45,13 @@ export default Component.extend({
...
-    if (!this.currentUser) {
+
+    if (!this.currentUser || !this.message) {
       return;
     }
-    this.element
-      .querySelector(".chat-message-content .chat-message-text")
+
+    document
+      .querySelector(`.chat-message-container[data-id='${this.message.id}']`)
```
Comment (CvX) [inline assets/javascripts/discourse/components/chat-message.js:54]:
```suggestion
    this.element
```

**Review by CvX — APPROVED** [review]

**Pull request merged**